common/src/config/game-support/ddr.ts

```diff
@@ -132,13 +146,21 @@ export const DDR_SP_CONF = {
 			description: "Flare Skill as it's implemented in DDR World.",
 			formatter: FmtScoreNoCommas,
 		},
+		exScore: {
+			description: "The EXScore.",
+			formatter: FmtScoreNoCommas,
+		},
 	},
 
 	sessionRatingAlgs: {
 		flareSkill: {
 			description: "Average of your 10 best Flare Points this session",
 			formatter: FmtScoreNoCommas,
 		},
+		exScore: {
+			description: "Average of your 10 best EXScores this session",
```

Comment:
I think this is a nonsensical way to rate sessions.

The amount of EX Score you can get on a chart is proportional to notes and not much else. This would just be measuring the average notecount of stuff you've played that session, which isn't really a meaningful signal.